File: app/src/main/java/com/pluu/webtoon/adapter/viewholder/DaumNoticeViewHolder.java
package com.pluu.webtoon.adapter.viewholder;

import android.content.Context;
import android.view.View;
import android.widget.TextView;

import com.pluu.webtoon.item.ChatView;

import butterknife.BindView;
import butterknife.ButterKnife;

/**
 * 공지 텍스트용 Item ViewHolder
 * Created by PLUUSYSTEM-SURFACE on 2016-05-21.
 */
public class DaumNoticeViewHolder extends BaseChattingViewHolder {
    @BindView(android.R.id.text1)
    TextView text1;

    public DaumNoticeViewHolder(View v) {
        super(v);
        ButterKnife.bind(this, v);
    }

    @Override
    public void bind(Context context, ChatView item) {
        text1.setText(item.getText());
    }
}

File: app/src/main/java/com/pluu/webtoon/adapter/viewholder/DaumNoticeViewHolder.kt
package com.pluu.webtoon.adapter.viewholder

import android.content.Context
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.pluu.webtoon.R
import com.pluu.webtoon.item.ChatView
import kotlinx.android.synthetic.main.view_chatting_notice_layout.view.*

/**
 * 공지 텍스트용 Item ViewHolder
 * Created by  pluu on 2017-05-02.
 */
class DaumNoticeViewHolder(v: View) : BaseChattingViewHolder(v) {

    override fun bind(context: Context, item: ChatView?) {
        if (item == null) return
        itemView.text1.text = item.text
    }

    companion object {
        fun newInstance(parent: ViewGroup) = DaumNoticeViewHolder(
                LayoutInflater.from(parent.context)
                        .inflate(R.layout.view_chatting_notice_layout, parent, false))
    }
}
